lib/gui/menu.js

```diff
@@ -84,13 +84,13 @@ const buildWindowMenu = (window) => {
         {
           label: 'Etcher Pro',
           click () {
-            electron.shell.openExternal('https://etcher.io/pro')
+            electron.shell.openExternal('https://etcher.io/pro?ref=etcher_menu')
```

Comment:
@tzikis what would be preferable here? `utm_source=etcher_menu`?

Reply:
@jhermsmeier as discussed in flowdock, i need `?utm_source=` for Mailchimp, but we can keep `?ref=` if we wanna use it on mixpanel

Reply:
@tzikis aye, I've added `utm_source` now, while also keeping the `ref` for Etcher Pro